hnn_core/network_builder.py

```diff
@@ -427,10 +409,11 @@ def _create_cells_and_feeds(self, threshold, record_vsoma=False,
             # how such 'common' spikes influence them
             else:
                 gid_idx = gid - self.net.gid_ranges[src_type][0]
-                et = self.net.feed_times[src_type][self.trial_idx][gid_idx]
-                feed_cell = _ArtificialCell(et, threshold, gid=gid)
-                _PC.cell(feed_cell.gid, feed_cell.nrn_netcon)
-                self._feed_cells.append(feed_cell)
+                et = self.net.external_drives[
```

Comment:
et as in event times? If so I would just be explicit.

Reply:
+1

Reply:
done